### DISCUSSION TITLE
Fails To Return To Tune Menu After Filament Change

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#798] [General Discussion]
Using Cura Filament Change plugin, set layer to change and process works until after "purge more" screen. Click "more" or "cancel" the next screen is the main menu instead of back to the tune menu. No more printing actions can be taken. The main menu works but no way to return to print tune menu. Pics show the sequence.

![Image01](https://user-images.githubusercontent.com/76499230/118000730-13edc700-b314-11eb-8f9c-b817685272a5.jpg)
![Image02](https://user-images.githubusercontent.com/76499230/118000735-14865d80-b314-11eb-95cc-5632b792d521.jpg)
![Image03](https://user-images.githubusercontent.com/76499230/118000739-14865d80-b314-11eb-90e3-b2eaf52565b7.jpg)
![Image04](https://user-images.githubusercontent.com/76499230/118000741-151ef400-b314-11eb-9d7a-00ebe8f51ce4.jpg)
![Image05](https://user-images.githubusercontent.com/76499230/118000742-151ef400-b314-11eb-8a7f-3ffe5279f24e.jpg)
![Image06](https://user-images.githubusercontent.com/76499230/118000748-16502100-b314-11eb-998e-db468311b472.jpg)

**Comment by Cal-LF** [comment]
Latest change "Temporary Purge More Fix" fixed this issue.

**Reply by tome9111991** [reply]
A other question.
How i can show layer height in status msg?
M117 not really works

**Reply by Cal-LF** [reply]
Use Cura post processing plugin "Display Filename and Layer on LCD.
Click display max layer change text to display to "Printing".
